src/main.py

```diff
@@ -193,12 +193,21 @@ def parse_flags(flags: List[str]) -> Options:
         "patterns are allowed.",
     )
     parser.add_argument(
+        # Deprecated, provided for backwards compatability. Use --data instead
         "--rodata",
         metavar="ASM_FILE",
-        dest="rodata_files",
+        dest="data_files",
         action="append",
         default=[],
-        help="read jump table data from this file",
+        help=argparse.SUPPRESS,
+    )
+    parser.add_argument(
+        "--data",
```

Comment:
an alternative to this btw would be supporting multiple .s files as input, where it would decompile all of them, and data/rodata files would just be .s files with no asm in them. we should perhaps discuss how we envision inter-function type inference working

Reply:
Yeah this is a great point. 

I could revert the changes to the CLI (except for help messages) if you think it's better to avoid changing the interface when we might change it again soon? Or do you think that should get resolved sooner?